homeassistant/components/device_tracker/owntracks.py

```diff
@@ -48,6 +51,16 @@ def owntracks_location_update(topic, payload, qos):
         if not isinstance(data, dict) or data.get('_type') != 'location':
             return
 
+        # Check for GPS accuracy
+        if CONF_MAX_GPS_ACCURACY in config:
```

Comment:
Probably best to set this  up in the scope of setup_scanner rather than in the callback function called on every update.